log4j-mongodb/src/main/java/org/apache/logging/log4j/mongodb/package-info.java

```diff
@@ -15,12 +15,13 @@
  * limitations under the license.
  */
 /**
- * The classes in this package contain the MongoDB provider for the NoSQL Appender.
+ * The classes in this package contain the MongoDB provider for the NoSQL
+ * Appender.
  */
 @Export
 @Open("org.apache.logging.log4j.core")
-@Version("2.20.1")
-package org.apache.logging.log4j.mongodb3;
+@Version("2.21.0")
```

Comment:
Let us use `2.24.0`, i.e. the version when the class was introduced. It is then easier to find the last Log4j version with some real changes to the module.

Reply:
Good catch!